app/src/app/admin/articles/article/post/post.tpl.html

```diff
@@ -21,7 +21,7 @@ <h2 ng-if="!ArticleController.$stateParams.newArticle">Edit</h2>
     <md-input-container flex>
         <label for="articleFormContent">Content</label>
 
-        <textarea id="articleFormContent" ng-model="ArticleController.article.content" columns="1"></textarea>
+        <textarea id="articleFormContent" ng-model="ArticleController.article.content" columns="1" ng-required="true"></textarea>
```

Comment:
No need to change this, but for future reference `ng-required="true"` is the same as `required`